src/server/sams/assets/service.py

```diff
@@ -64,6 +71,12 @@ def patch(self, item_id: ObjectId, updates: Dict[str, Any]) -> Dict[str, Any]:
         content = updates.pop('binary', None)
         self.validate_patch(original, updates)
 
+        external_user_id = get_external_user_id(updates)
+
+        if external_user_id:
+            updates['versioncreated'] = utcnow()
```

Comment:
`versioncreated` should always be set, regardless if `external_user_id` is supplied or not